src/validations/planBasics.ts

```diff
@@ -0,0 +1,45 @@
+import * as Yup from 'yup';
+
+const datePickerSchema = Yup.date()
+  .nullable()
+  .min(new Date(), 'Date cannot be in the past');
+
+const planBasicsSchema = {
+  pageOneSchema: Yup.object().shape({
+    modelName: Yup.string().trim().required('Enter the Model Name'),
+    modelCategory: Yup.string().required('Enter the Model Category'),
+    cmsCenters: Yup.array()
+      .min(1, 'Select a CMS Component')
+      .required('Select a CMS Component')
+  }),
+
+  pageOneSchemaWithCmmiGroup: Yup.object().shape({
+    modelName: Yup.string().trim().required('Enter the Model Name'),
+    modelCategory: Yup.string().required('Enter the Model Category'),
```

Comment:
This should be - `modelCategory: Yup.string().nullable().required('Enter the Model Category')`